.gcloudignore

```diff
@@ -0,0 +1,8 @@
+# Default .gcloudignore file is:
```

Comment:
Why do we need it ?

Reply:
If it does not exist, it is created, with the default values exposed here as comments. So we are creating ones with the default values minus `.git`

Reply:
So we're checking the default for all images, not just prodbox. Can we leave the default and only pass an override when we build prodbox 🙏 ?